<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elevation Profile Catcher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Layout_main" stretch="3,3,1">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="styleSheet">
        <string notr="true">border-color: rgb(32, 32, 32);</string>
       </property>
       <property name="title">
        <string>Station 1</string>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>489</width>
          <height>81</height>
         </rect>
        </property>
        <property name="title">
         <string>Latitude</string>
        </property>
        <widget class="QSpinBox" name="spinBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>102</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_2">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>30</y>
           <width>102</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>30</y>
           <width>122</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>61</width>
           <height>33</height>
          </rect>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>30</y>
           <width>21</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>'</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>30</y>
           <width>21</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&quot;</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>481</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>Longitude</string>
        </property>
        <widget class="QSpinBox" name="spinBox_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>102</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_4">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>30</y>
           <width>102</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>30</y>
           <width>122</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>61</width>
           <height>33</height>
          </rect>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>30</y>
           <width>21</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>'</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>30</y>
           <width>21</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&quot;</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Station 2</string>
       </property>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>489</width>
          <height>81</height>
         </rect>
        </property>
        <property name="title">
         <string>Latitude</string>
        </property>
        <widget class="QSpinBox" name="spinBox_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>102</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_6">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>30</y>
           <width>102</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>30</y>
           <width>122</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_3">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>61</width>
           <height>33</height>
          </rect>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>30</y>
           <width>21</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>'</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>30</y>
           <width>21</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&quot;</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>481</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>Longitude</string>
        </property>
        <widget class="QSpinBox" name="spinBox_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>102</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_8">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>30</y>
           <width>102</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>30</y>
           <width>122</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_4">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>61</width>
           <height>33</height>
          </rect>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>30</y>
           <width>21</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>'</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>30</y>
           <width>21</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&quot;</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
